iset.mm

```diff
@@ -67983,6 +67983,135 @@ property of disjoint unions (see ~ updjud in the case of functions).
+      CLZUVTNZUDZCJUEZUFZDMZUVQUVSBEJUVOUVPUVRUGUHUVSUWIEFLZUVROZPAUIZNZFJUJZJU
+      CZUVSUWNAUVPUBZBMZUWDUWNNZUDZCJUEZUWIUWMFJUKUVSUWNAPULZUVRUMZUBZUWQUVSUWN
+      AUXBUNZGLZHLZUXBOZQZHUWNUOZGAUEUXCUVSUXBUWNUPZUXEUXBOZANZGUWNUEUXDUVSUXBU
+      QZUXBURZUWNQUXJUVSUXAUQZUVRUQZUXMRUSUTRVAZPVBZUXQRUXAVBUXOVCRUXQPVDUXQRUX
+      AVEVFZJUVOUVRVGZUXAUVRVHVIUVSCUXNUWNUVSUWDUXNNZUWDJNZUWDUVROZUWLNZSZUWRUV
+      SUWDUVRURZNZUYCUXAURZNZSZUYBUYISUYAUYEUVSUYGUYBUYIUVSUYFJUWDUVSJUVOUVRJUV
+      OUVRVKZVLVJVMUVSUXPUYAUYJVNUXTUYAUYGUYASUXPUYJUYAUYGUXNUYFUWDUXAUVRVOVPVQ
+      UXPUYGUYAUYIUWDUXAUVRVRVSVTWCUVSUYBUYDUYIUVSUYBSZUYDUYDUYIVNUYCWAKUIZNZUY
+      LUYDSZUYDUYIUYLUYDWBUYOUYCPWDZNZUYIUYDUYQUYLUWLUYPUYCPAWEVPTUYPUYHUYCPWFW
+      GZWHWIUYLUYNSZUYDUYIUYNUYDWPZUYLUYDUYNUWLUYMWJUSQUYDUYNWPZAKWKUWLUYMUYCWL
+      WMWNZTUYSUYQUYIUYNUYQWPUYLUYQUYNVUAUYCPRUIZUYPVUCUYMWJUSQUYCVUCNVUARKWKVU
+      CUYMUYCWLWMPWOWQWNTUYRWRWSUYLUYCUWLUYMWTZNZUYDUYNXGZUYLUYCUVONZVUEUVSJUVO
+      UWDUVRUYKXAZVUDUVOUYCAKXBWGZXCUYCUWLUYMXDZWHXEVSXFUWMUYDFUWDJFCXHUWKUYCUW
+      LUWJUWDUVRXIXJXKZXSXLUXBUWNXMXNUVSUXLGUWNUVSUXEUWNNZSZUXKUXEUVROZUXAOZAVU
+      MJUVOUVRUNZUXEJNZUXKVUOQUVSVUPVULUYKXOVUMVUQVUNUWLNZVULVUQVURSZUVSVULVUSU
+      WMVURFUXEJFGXHUWKVUNUWLUWJUXEUVRXIXJXKXPTZXQJUVOUXEUXAUVRXRXTVUMILZPOZVUN
+      QZVUOANIAVUMVUQVURVVCIAUOZVUTPUQZVURVVDUXRVVEVCRUXQPVEYAZIVUNAPYBWMYCVUMV
+      VAANZVVCSSZVVBUXAOZVUOAVVHVVBVUNUXAVUMVVGVVCYDYEVVHVVIVVAAUXRVVARNVVIVVAQ
+      VCIYFRUXQVVAPYGYIVUMVVGVVCYHYJYKYLYMYNGUWNAUXBYOXNUVSUXIGAUVSUXEANZSZUXEP
+      OZUXFUVROZQZHUWNUOZUXIVVKUXFUWNNZVVNSZHMZVVOVVKUXFJNZVVNSZHMZVVRVVJUVSVVL
+      UVONZVWAAKUXEUUAUVSVWBSVVNHJUOVWAHJUVOVVLUVRUUBVVNHJYPWHUUCVVKVVTVVQHVVKV
+      VTVVQVVKVVTSZVVPVVNVWCUWMVVMUWLNFUXFJFHXHUWKVVMUWLUWJUXFUVRXIXJVVKVVSVVNY
+      HVWCVVLVVMUWLVVKVVSVVNYDZVVJVVLUWLNZUVSVVTVVEUXEPURZNVVJVWEUUDVVFUXERVWFG
+      YFZUXRVWFRQVCRUXQPUUEYAUUFAUXEPUUGYIUUHYKUUIVWDUUJYQUUKYSVVNHUWNYPXCVVKVV
+      NUXHHUWNVVKVVPSZVVNUXHVWHVVNSZUXEVVMUXAOZUXGVWIUXEVVLUXAOZVWJUXRUXERNVWKU
+      XEQVCVWGRUXQUXEPYGYIVWIVVLVVMUXAVWHVVNWBYEUULVWHUXGVWJQZVVNVWHVUPVVSVWLUV
+      SVUPVVJVVPUYKUUMVVPVVSVVKUWMFUXFJUUNTJUVOUXFUXAUVRXRXTXOUUOYQUUPYSYNHGUWN
+      AUXBUUTXNUWPUXCBUXBUXOUVRRNUXBRNUXSEYFUXAUVRRUUQYIUWNAUVPUXBUGYRWCUVSUWSC
+      JUYLUYDUDZUWSUYLUYDUYTXGZVWMUYLVUFVWNUYLVUGVUFVUHVUGVUEVUFVUIVUJUURWHUYNU
+      YTUYDVUBUUSWCUYDUVAXCUYLUYDUWRUYLUYDUYEUWRUYBUYDUYEVNUVSUYBUYDUVBTVUKXSYT
+      UVCYNUWHUWOUWQUWTUFDUWNUWMFJUVKUVDUVTUWNQZUWAUWOUWCUWQUWGUWTUVTUWNJUVEVWO
+      UWBUWPBUVTUWNAUVPUVFUVGVWOUWFUWSCJVWOUWEUWRUVTUWNUWDUVLYTUVHUVIYRUVJUVMUV
+      N $.
+  $}
+
+  ${
+    $d A f g m s $.  $d A f g x $.  $d A f m s y z $.  $d f m n s y z $.
+    $d A n $.
+    $( A set is countable iff there is a surjection from a decidable subset of
+       the natural numbers onto it.  The decidability condition is needed as
+       shown at ~ ctssdc .  (Contributed by Jim Kingdon, 15-Aug-2023.) $)
+    ctssdc $p |- ( E. s ( s C_ _om /\ E. f f : s -onto-> A
```

Comment:
the theorem is quoting itself? + maybe you meant ctssexmid below

(approve when done)

Reply:
Yes, it is supposed to be ctssexmid

Thanks for noticing this.